utils/tests/test_run_condordance_docker.py

```diff
@@ -0,0 +1,97 @@
+        test_data_path = os.path.split(os.path.split(current_path)[0])[0]
+        user_name = os.path.expanduser('~')
+
+        rel_path_to_hg38_ref = 'dev/hg38/hs38DH.fa'
+
+        self.reference = os.path.join(os.sep, user_name, rel_path_to_hg38_ref)
+        self.tsv_file = os.path.join(
+            os.sep, test_data_path,
+            'test_data/concordance_output_sample.tsv')
+        self.vcf_chr17 = os.path.join(
+            os.sep, test_data_path,
+            'test_data/chr17_1_83257441_paste.sites.vcf.gz')
+        self.vcf_chr04 = os.path.join(
+            os.sep, test_data_path,
+            'test_data/chr4_1_190214555_paste.sites.vcf.gz')
+        self.vcf_chr04_compromised = os.path.join(
+            os.sep, test_data_path,
+            'test_data/chr4_1_190214555_paste_SNPs_del.sites.vcf.gz')
+        self.output_path = os.path.join(os.sep, test_data_path,
+                                        'test_data/test_output.tsv')
+
+    def tearDown(self):
+        pass
+
+    def test_process_output_tsv(self):
+        d = process_output_tsv(self.tsv_file)
+        pprint(d)
+
+    def test_list2dict(self):
+        L = []  # list to capture results
+        with open(self.tsv_file, newline='') as csvfile:
+            file_reader = csv.reader(csvfile, delimiter=' ', quotechar='|')
+            for row in file_reader:
+                L.append(row[0])
+        d = list2dict(L)
+
+        self.assertEqual(d['type']['SNP']['true-positive'], '12')
+        self.assertEqual(d['type']['INDEL']['sensitivity'], 'NaN')
+
+    def test_run_concordance_same_file(self):
+        """Testing perfect overlap by using the same file as truth and test."""
+        run_concordance(reference=self.reference,
+                        eval_file=self.vcf_chr17,
+                        truth_file=self.vcf_chr17,
+                        output_file=self.output_path)
+        tf = process_output_tsv(self.output_path)
+
+        self.assertEqual(tf, 0)
+
+    def test_run_concordance_different_file(self):
+        """Challenging the method by using files from two different chromosomes,
+        which likely have zero overlap. This test should fail."""
+        run_concordance(reference=self.reference,
+                        eval_file=self.vcf_chr17,
+                        truth_file=self.vcf_chr04,
+                        output_file=self.output_path)
+        tf = process_output_tsv(self.output_path)
+
+        self.assertEqual(tf, 1)
+
+    def test_run_concordance_SNPs_del(self):
+        """Challenging the method by using two files from the same chromosome,
+        but in one of them 20% of SNPs have been deleted. Should fail."""
+        run_concordance(reference=self.reference,
+                        eval_file=self.vcf_chr04_compromised,
+                        truth_file=self.vcf_chr04,
+                        output_file=self.output_path)
+        tf = process_output_tsv(self.output_path)
+
+        self.assertEqual(tf, 1)
```

Comment:
This also fails and says that they are concordant when I run the tests on my machine.